Gems/ROS2/Code/Source/VehicleDynamics/VehicleInputsState.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Time/ITime.h>
+#include <type_traits>
+//#include <concepts> //C++20
```

Comment:
We could remove the commented out c++20 items here and create a backlog GHI to incorporate it when we update to C++20 for the engine